app/models/message.rb

```diff
@@ -365,4 +367,10 @@ def set_conversation_activity
     conversation.update_columns(last_activity_at: created_at)
     # rubocop:enable Rails/SkipsModelValidations
   end
+
+  def update_message_sentiments
+    # override in the enterprise ::Enterprise::SentimentAnalysisJob.perform_later(self)
+  end
```

Comment:
@tejaswinichile don't have much context on this, but is it possible to not have this method here at all? CC: @sojan-official